Example Apps/Example Swift/MenuManager.swift

```diff
@@ -206,6 +207,53 @@ private extension MenuManager {
             })
         })
     }
+    
+    /// Menu item that shows remote control example
+    ///
+    /// - Parameter manager: The SDL Manager
+    /// - Returns: A SDLMenuCell object
+    class func menuCellRemoteControl(with manager: SDLManager) -> SDLMenuCell {
+        
+        /// Initialize Remote Control Manager
+        let remoteControlManager = RemoteControlManager(sdlManager: manager)
```

Comment:
Like suggestions in Obj-C, this should be held and initialized elsewhere, otherwise at the end of this function scope it's gonna disappear.